lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
I made a quick fiddle which places button inside a dropdown https://jsfiddle.net/q524gxh7/

So it would be an easy component to make.  It just needs to emit a click event upwards, and accept the disabled prop.